docker/compose.deploy.yaml

```
@@ -223,8 +237,26 @@ services:
+    restart: unless-stopped
+    volumes:
+      - source: redis_data
+        target: /data
+        type: volume
+
 volumes:
   caddy_config:
   caddy_data:
   minio_data:
   postgres_data:
+  redis_data:
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)